src/test/java/uk/gov/hmcts/reform/managecase/api/controller/NoticeOfChangeControllerTest.java

```diff
@@ -534,8 +537,202 @@ void shouldReturnSuccessfullyIfApprovalStatusIsNotApproved() throws Exception {
         @DisplayName("should error if case reference in Case Details is empty")
         @Test
         void shouldFailIfCaseReferenceIsEmpty() throws Exception {
+            caseDetails = new CaseDetails(null, "Jurisdiction", "State", "CaseTypeId", new HashMap<>());
```

Comment:
Uses TestFixtures were possible throughout these unit tests.  e.g. `defaultCaseDetails().id(null).build();`

As this is more ACA-85 code `check-noc-approval` is this another regression?